website/content/docs/nia/configuration.mdx

```diff
@@ -330,6 +330,14 @@ driver "terraform-cloud" {
   - We recommend creating a dedicated team and team API token to isolate automation by Consul-Terraform-Sync from other Terraform Cloud operations.
 - `workspace_prefix` - (string) Specifies a prefix to prepend to the automatically-generated workspace names used for automation. This prefix will be used by all tasks that use this driver. By default, when no prefix is configured, the workspace name will be the task name. When a prefix is configured, the workspace name will be `<workspace_prefix value><task name>`. For example, if you configure the prefix as "cts_", then a task with the name "task_firewall" will have the workspace name "cts_task_firewall".
 - `required_providers` - (obj: required) Declare each Terraform provider used across all tasks. This can be configured the same as how you would configure [Terraform `terraform.required_providers`](https://www.terraform.io/docs/configuration/provider-requirements.html#requiring-providers) field to specify the source and version for each provider. Consul-Terraform-Sync will process these requirements when preparing each task that uses the provider.
+- `tls` - Configure TLS to allow HTTPS connections to [Terraform Enterprise](https://www.terraform.io/docs/enterprise/install/installer.html#tls-key-amp-cert).
+  - `enabled` - (bool) Enable TLS. Specifying any option for TLS will also enable it.
```

Comment:
@findkim Can we be a bit more specific? It's not intuitive what "specifying any options" means.

Reply:
Setting `verify`, `key`, `ca_cert`, `ca_path`, `cert`, or `server_name` to any value would all enable the TLS checking.

Reply:
Gotcha,  maybe we can state something like "By providing a value for any of the TLS  option will also enable this parameter implicitly. 

```suggestion
  - `enabled` - (bool) Enable TLS. By providing a value for any of the TLS options will enable this parameter implicitly.
```

Reply:
Thanks! I like your iteration of this.